src/components/image-contentBlock/image-contentBlock.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+import { down } from '../breakpoint/breakpoint';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+
+const BlockWrapper = styled.div<{
+  textAlign: 'right' | 'left' | 'bottom';
```

Comment:
I actually thought about that, I have tried the "interface-way" in another component, and was thinking if I like the way with less variables better, but I have implemented it using an interface

Reply:
Less is not better here as it makes reading the code more difficult because you mix two very different things. Usually we have defined rules that ideally are enforced through the linter for things like that, so your machine would complain. Right now we don't have any rule, so let's see how you think about that the next time.

Reply:
👍